app/services/proxy_status/logs.rb

```diff
@@ -0,0 +1,18 @@
+module ProxyStatus
+  class Logs < Base
+    def api
+      ProxyAPI::Logs.new(:url => proxy.url)
```

Comment:
Oh actually this and above is necessary because the class is named `Logs` while `humanize` makes it `Log` causing a name error. Since the REST API is also named `/logs`, I'd rather keep it as is.